generate-stackbrew-library.sh

```diff
@@ -57,7 +57,8 @@ cat <<-EOH
 # this file is generated via https://github.com/docker-library/ghost/blob/$(fileCommit "$self")/$self
 
 Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
-             Joseph Ferguson <yosifkit@gmail.com> (@yosifkit)
+             Joseph Ferguson <yosifkit@gmail.com> (@yosifkit),
+             Austin Burdine (@acburdine)
```

Comment:
Also happy to include the email address from your commits here, if you're OK with that, but didn't want to include it presumptuously since it's not publicly listed on your GitHub profile. :+1:

Reply:
(It would also be totally appropriate for us to list you _first_ instead of last, but I don't want to push you too hard! :smile:)

Reply:
haha this is fine 😄 - feel free to add `austin@acburdine.me` as the email